Sources/XcodeGenKit/ProjectGenerator.swift

```diff
@@ -96,6 +96,72 @@ public class ProjectGenerator {
                         archiveAction: archiveAction)
     }
 
+    func generateBreakpointList(_ breakpoints: [Breakpoint]) throws -> XCBreakpointList {
+        return XCBreakpointList(type: "4", version: "2.0", breakpoints: try breakpoints.flatMap({ try generateBreakpointProxy($0) }))
+    }
+
+    private func generateBreakpointProxy(_ breakpoint: Breakpoint) throws -> XCBreakpointList.BreakpointProxy {
+        var extensionID: XCBreakpointList.BreakpointProxy.BreakpointExtensionID
+        switch breakpoint.extensionID {
```

Comment:
Could Breakpoint.extensionID just be a `XCBreakpointList.BreakpointProxy.BreakpointExtensionID`, similar to `Target.type`?